src/rules/requireJsdoc.js

```diff
@@ -0,0 +1,166 @@
+//
+// The requireJDDoc rule at eslint is deprecated and won't get any bugfixes anymore.
+// Thus this method implements a compatible requireJSDoc rule.
+//
+
+/* eslint-disable sort-keys */
```

Comment:
Unnecessary eslint-disable/ comment.

Sorted property style is enforced across the entire library. This rule implementation is not an exception.